lib/twine/version.rb

```diff
@@ -1,3 +1,3 @@
 module Twine
-  VERSION = '0.10.1'
+  VERSION = '0.10.2'
```

Comment:
I prefer to update the version myself in a special commit when it is time to version twine. Mind taking this out?